src/components/Swap/Toolbar/index.tsx

```diff
@@ -194,14 +206,14 @@ export default memo(function Toolbar() {
   return (
     <StyledExpando
       title={
-        <ToolbarRow flex justify="space-between" data-testid="toolbar" onClick={onToggleOpen}>
+        <ToolbarRow flex justify="space-between" data-testid="toolbar" onClick={maybeToggleOpen}>
```

Comment:
onClick is handled in the parent (for the whole row), so it is unnecessary (and subverts the DOM model) on Trade and PriceImpact. Remove it there.